src/System.Globalization/tests/RegionInfo/RegionInfoTests.Properties.cs

```diff
@@ -51,8 +51,8 @@ public void NativeName(string name, string expected)
         [Theory]
         [InlineData("en-US", new string[] { "United States" })]
         [InlineData("US", new string[] { "United States" })]
-        [InlineData("zh-CN", new string[] { "China", "People's Republic of China" })]
-        [InlineData("CN", new string[] { "China", "People's Republic of China" })]
+        [InlineData("zh-CN", new string[] { "China", "People's Republic of China", "China mainland" })]
```

Comment:
are we going to always run on this specific version of MacOS? should we check the ICU version and behave accordingly?

Reply:
> are we going to always run on this specific version of MacOS

I don't think its only for this specific version, as same update exist in master, i guess it just was not ported to 2.x: https://github.com/dotnet/corefx/blob/master/src/System.Globalization/tests/System/Globalization/RegionInfoTests.cs#L95, just making same update

Reply:
What I mean is if you run on different OSX version which could be using a different version of ICU. This will make the test fail there I guess.

Reply:
Seems this new string is added to the new version of ICU and other strings represent previous ICU value, so for other/previous versions of ICU it will not fail because test is checking if the result string is existing within this string array:
```csharp
public void EnglishName(string name, string[] expected)
{
    string result = new RegionInfo(name).EnglishName;
    Assert.True(expected.Contains(result));
}
```

Reply:
we should be fine then. Thanks.